CONTRIBUTING.md

```diff
@@ -0,0 +1,35 @@
+# Contributing to Grafana k6 Studio
+
+Thank you for your interest in contributing to Grafana k6 Studio! We welcome all people who want to contribute in a healthy and constructive manner within our community. To help us create a safe and positive community experience for all, we require all participants to adhere to the [Code of Conduct](.github/CODE_OF_CONDUCT.md).
+
+If you want to chat with the team or the community, you can [join our community forums](https://community.grafana.com/c/grafana-k6/k6-studio/)
+
+## Report issues
+
+Before submitting a new issue, try to make sure someone hasn't already reported the problem. Look through the [existing issues](https://github.com/grafana/k6-studio/issues) for similar issues.
+
+Report a bug by submitting a [bug report](https://github.com/grafana/k6-studio/issues/new?template=bug.yaml).
+
+Follow the issue template and add additional information that will help us replicate the problem.
+
+## Contribute to code
+
+> [!NOTE]
+> Grafana k6 Studio is currently in active development. Most of the features are meant to be implemented by the team, so we may or may not accept contributions from the community.
+> Non-critical bug fixes and small features are welcome, but please check with us first to see if we are already working on it.
```

Comment:
```suggestion
```

Reply:
My intention was to set the right expectations and make the list that follows shorter than in the k6 doc. I agree that the tone can be improved though.